### ISSUE TITLE
RNN-Transducer Loss support

### ISSUE THREAD

**Issue by Times125** [tensorflow/addons#641]
I found that there are no RNN-Transducer  loss in the new version. I have to use third party pkg like [wrap-transducer](https://github.com/HawkAaron/warp-transducer/tree/master/tensorflow_binding)
So, official support in both tf1.x and tf2 is therefore expected. Thanks！

**Comment by seanpmorgan** [comment]
I'm not very familiar with this so I'll defer to @qlzh727 as subpackage owner to evaluate if this makes sense within addons.

**Comment by qlzh727** [comment]
Sorry for the very late reply.

I am not very familiar with that too. Could u provide a reference to the paper for it? 

Also, for the addon package, we are targeting TF 2.0, which means 1.x is probably not supported.

**Comment by WindQAQ** [comment]
Paper is here https://arxiv.org/abs/1211.3711. It's quite similar to CTC loss.

**Comment by will-rice** [comment]
This issue is pretty old, but since it is still open does that mean I could work on an implementation? I work on ASR and would benefit from an implementation with TensorFlow ops. From what I can tell, everyone is still using the warp-transducer version, and the build process adds additional complexity to projects.

**Comment by qlzh727** [comment]
Sure, any contribution is welcome.

**Comment by iamjanvijay** [comment]
@will-rice I have an implementation of RNNT ([link to the repository](https://github.com/iamjanvijay/rnnt)) using TensorFlow ops.  We can work on this together, if you are interested?

**Comment by will-rice** [comment]
@iamjanvijay sounds good!

**Comment by will-rice** [comment]
@iamjanvijay do you want me to just port your ops to the addons loss format and tag you in the PR?

**Comment by iamjanvijay** [comment]
@will-rice Yes. That can be a good way to start then we can do iterations over it. What do you suggest?

**Comment by will-rice** [comment]
> @will-rice Yes. That can be a good way to start then we can do iterations over it. What do you suggest?

that works. I'll put it together and then tag you.

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)

**Comment by Times125** [comment]
您好，我已经收到您的来信，会尽快处理。
